internal/resolver/dns/dns_resolver.go

```diff
@@ -41,9 +41,15 @@ import (
 	"google.golang.org/grpc/serviceconfig"
 )
 
-// EnableSRVLookups controls whether the DNS resolver attempts to fetch gRPCLB
-// addresses from SRV records.  Must not be changed after init time.
-var EnableSRVLookups = false
+var (
```

Comment:
Could you please also pull in `ResolvingTimeout` here?

Reply:
Thanks for the comments! I also pulled the `logger` variable. Let me know if it was not a good idea.